index.html

```diff
@@ -5,330 +5,360 @@
+                <div class="ptext">
                 </div>
+                <div class="containerbasic"></div>
+            </div>
+        </section>
+        <section class="about">
+            <div class="aboutcontainer">
+                <div class="partone">
+                    <div class="iamtext2">
+                        <div class="primarytext">
+                            <h2>About Myself</h2>
+                        </div>
+                        <div class="subtext">
+                            <p> Hello,I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                            </p>
+                        </div>
+                        <div class="socialmedia">
+                            <p>LET'S CONNECT</p>
+                        </div>
 
-                <div class="holder2">
-                    <div id="lets2" class="let2">
-                        <p>LET'S CONNECT</p>
+                        <div class="mediaicons">
+                            <ul>
+                                <li><img src="./Images/twitter.svg" alt="T"></li>
+                                <li><img src="./Images/linkedin.svg" alt="L"></li>
+                                <li><img src="./Images/medium.svg" alt="M"></li>
+                                <li><img src="./Images/github.svg" alt="G"></li>
+                                <li><img src="./Images/angellist.svg" alt="A"></li>
+                            </ul>
+                        </div>
+                        <div class="resume">
+                            <button class="btntext">Get My Resume</button>
+                        </div>
                     </div>
-                    <ul class="icon2">
-
-                        <li class="first">
-                            <a href="first"><img src="./Images/twitter.svg" alt="twitter"></a>
-                        </li>
-                        <li class="second">
-                            <a href="second"><img src="./Images/Linkedin icon.svg" alt="Linkedin"></a>
-                        </li>
-                        <li class="third">
-                            <a href="third"><img src="./Images/github.svg" alt="github"></a>
+                </div>
+                <div class="subcontainer">
+                    <div class="languages">
+                        <a href="#" class="langcontainer">
+                            <h4>Languages</h4>
+                        </a>
+                    </div>
+                    <ul class="last">
+                        <li class="js">
+                            <a href="#"><img class="EJS" src="./Images/Frame 9.png" alt="#">
+                            </a>
```

Comment:
- [ ] On the mobile screen, each technology card content should be on the same axis horizontally. kindly follow the design guidelines as much as possible (e.g., font, colors, images, tests, margins) 
![image](https://user-images.githubusercontent.com/33767951/140236517-4b440516-a180-4d07-8fdb-efb83c7e83f3.png)